sources/Inputs/include/Behaviours.h

```diff
@@ -72,5 +73,43 @@ struct Generator {
   double pmax;                             ///< maximum active power
 };
 
+class HvdcLine;
```

Comment:
Isn't it possible to avoid those 2 classes to reference each-other? It is always good to avoid such patterns

Reply:
For example we could have the ids of the converters and their bus ids?

Reply:
Are you talking about the dfl::inputs::HvdcLine or the dfl::algo::HvdcLineDefinition ?

Reply:
HvdcLine and Converters

Reply:
For dfl::algo::HvdcLineDefinition, not factorizing the attributes would result in having in having 6 attributes compared to the 2 before, do we want that or a new structure ?

Reply:
it would be conerter ids, bus ids and?

Reply:
And the voltageRegulationOn for the .par

Reply:
on my side I am ok with adding the attributes

Reply:
Or I could add a new Structure in dfl::algo : ConverterDefinition ?

Reply:
yes why not

Reply:
multiple encapsulation levels are not always a good idea. In that case, it is not a bug deal that it has more attributes. You can put in comments its origins or not if it iis obvious enough by the constructor

Reply:
Ok, this is done